kinisot/Hess_to_Freq.py

```
@@ -3,95 +3,50 @@
+   # Swap the requested atoms (1-based numbers in the comma-separated iso
+   # string; '0' means no substitution) for their heavier isotopes.
+   # Substitution considers 1H/2H, 12C/13C and 16O/17O. Elements are
+   # identified by a mass window so that both Gaussian's isotopic masses
+   # (12.00000) and ORCA's average atomic masses (12.011) are recognized.
+   # More can be added, but it hasn't been necessary so far...
+   mass_list = list(mass_list)
+   for atom in iso.split(','):
+      i = int(atom) - 1
+      if i < 0 or i >= len(mass_list):
+         continue
+      if 0.9 < mass_list[i] < 1.5: mass_list[i] = 2.0141    # 1H - 2D
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)